Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -17644,6 +17632,84 @@
         <SearchFlags index="ExtendByUnitRadius" value="1"/>
         <SearchFlags index="OffsetByUnitRadius" value="1"/>
     </CEffectEnumArea>
+    <CEffectCreatePersistent id="AP_ReleaseBombersPatrolCP">
+        <WhichLocation Value="CasterUnit"/>
+        <EditorCategories value="Race:Protoss"/>
+        <Flags index="RandomEffect" value="1"/>
+        <Flags index="RandomOffset" value="1"/>
+        <PeriodCount value="1"/>
+        <PeriodicEffectArray value="ReleaseBombersPatrolLeftTurnCP"/>
+        <PeriodicEffectArray value="ReleaseBombersPatrolRightTurnCP"/>

```

Comment:
These effects don't exist

Reply:
Fixed, copied the effects over from starcoop gamedata. Didn't look too closely to see if interceptor movement effects were reusable, but that didn't seem to be the case at a glance.